engine/runtime/src/main/scala/org/enso/compiler/Passes.scala

```diff
@@ -56,6 +56,8 @@ class Passes(
     List(
       ExpressionAnnotations,
       AliasAnalysis,
+      MethodReferences,
+      AliasAnalysis,
```

Comment:
Off-topic: Shouldn't these passes be collected via `ServiceLoader.load`? Then the system would become modular & extensible. Individual passes could be separated into own compilation units. Some of them could be skipped (during tests or even in runtime).